### DISCUSSION TITLE
How to set proxy options in http connection

### DISCUSSION THREAD

**Discussion by trlopes1974** [apache/airflow#37427] [Q&A]
Hi

I'm having trouble to get this working.
I have an http connection set where the extra filed has:

"extra": {
    "Authorization": "Bearer KFDGKDFGJFDKJGDKJG",
    "content-type": "application/json",
    "proxies": "{\"http\": \"http://100.100.100.100:3128\",\"https\": \"http://100.100.100.100:3128\"",
    "verify": "false"
  },

But I keep getting timeouts as the connection does not go trough the proxy.

ahs the above formating is messed up, here is what I have in the connection defenition.
![image](https://github.com/apache/airflow/assets/16724800/ef5b3212-4a2b-46f8-bc4f-22a0e141e695)

**Comment by rinaldistefano** [comment]
Goodmorning,
any good news about this topic ?
Thank you
regards
